app/controllers/stories_controller.rb

```diff
@@ -28,6 +28,14 @@ def show
     render json: @story
   end
 
+  def sort
+    ordered_ids = params[:ordered_ids]
```

Comment:
You don't need this variable here, just call pass `params[:ordered_ids]` to `SortStores`. It's fine.